extensions/clipboard/ace_clipboard.cpp

```diff
@@ -30,44 +33,49 @@ void __stdcall RVExtension(char *output, int outputSize, const char *function) {
     }
 
     if (!strcmp(function, "version")) {
-        strncpy(output, ACE_FULL_VERSION_STR, outputSize);
+        strncpy_s(output, outputSize, ACE_FULL_VERSION_STR, _TRUNCATE);
         EXTENSION_RETURN();
     }
 
-#ifdef _WIN32
-
-    if (!strcmp(function, "--COMPLETE--")) {
-        HGLOBAL hClipboardData = GlobalAlloc(GMEM_FIXED, gClipboardData.length() + 1);
-        if (!hClipboardData) {
-            result = "GlobalAlloc() failed, GetLastError=" + GetLastError();
-            gClipboardData = "";
-            EXTENSION_RETURN();
-        }
-
-        char *pClipboardData = (char *)GlobalLock(hClipboardData);
-        if (!pClipboardData) {
-            result = "GlobalLock() failed, GetLastError=" + GetLastError();
-            gClipboardData = "";
-            EXTENSION_RETURN();
-        }
-        memcpy(pClipboardData, gClipboardData.c_str(), gClipboardData.length());
-        pClipboardData[gClipboardData.length() + 1] = 0x00;
-
-        GlobalUnlock(hClipboardData);
+    #ifndef _WIN32
+        EXTENSION_RETURN();

```

Comment:
Only if the pre-processor definition _WIN32 is undefined, so basically only if compiled/used on linux.